drafts/concise_guide.ipynb

```diff
@@ -161,6 +220,22 @@
     "```"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Enums can be used to constrain the values accepted for a route parameter.\n",
+    "\n",
+    "```python\n",
+    "ModelName = str_enum('ModelName', \"alexnet\", \"resnet\", \"lenet\")\n",
+    "\n",
```

Comment:
We said that query params are idiomatic so let's use them.

Reply:
We're confused, by query parameters do you mean the key/value that come after the question mark?

`/models?nm=alexnet`

Or do you mean this kind of parameter?

`/models/alexnet`

Reply:
We figured it out. We demonstrate query parameters later.